src/main/java/org/opentest4j/ValueWrapper.java

```diff
@@ -122,6 +124,14 @@ public int getIdentityHashCode() {
 		return this.identityHashCode;
 	}
 
+	/**
+	 * Returns the value supplied to {@link #create(Object)}. if {@code ValueWrapper}
+	 * has been been serialized this will return {@code null}
```

Comment:
Please add a blank line after the first sentence and start the new paragraph with `<p>`.

Reply:
Can do, thanks for the feedback!

Reply:
`@since` is missing

Reply:
Set to `1.2` as that's the current snapshot version listed in master since last release 👍